ImageCapture/webcam_image_capture.py

```diff
@@ -0,0 +1,46 @@
+import cv2
+import os
+import time
+from datetime import datetime
+
+# Paramètres
+interval_seconds = 5  # Intervalle de temps entre chaque capture en secondes
+save_directory = "captured_images"  # Dossier où les images seront enregistrées
+
+# Création du dossier pour stocker les images, s'il n'existe pas
+if not os.path.exists(save_directory):
+    os.makedirs(save_directory)
+
+# Initialisation de la caméra (0 pour la caméra par défaut, essayez 1 si vous utilisez une autre webcam)
+cap = cv2.VideoCapture(0)
+
+if not cap.isOpened():
+    print("Erreur: Impossible d'accéder à la webcam.")
+    exit()
+
+try:
+    print("Press Ctrl+C to stop capturing images.")
+    while True:

```

Comment:
I wonder if there is a better way to periodically call this function. `while True` loops and `time.sleep` can make unexpected behaviors in some cases.